RPG Wizardry/Assets/Scripts/Player/Movement/MovementManager.cs

```diff
@@ -50,6 +50,16 @@ public void Stun(float duration)
         {
             StartCoroutine(StunLoop(duration));
         }
+
+        public void ToggleMovement(bool value)

```

Comment:
This method does not Toggle Movement. It Enables/Disables it based on a parameter 😉 .
Either change the name, or change the function such that it toggles instead (no param).